apps/lib/p5play/p5.play.js

```diff
@@ -1543,8 +1558,11 @@ function Sprite(pInst, _x, _y, _w, _h) {
       imageMode(CENTER);
 
       translate(this.position.x, this.position.y);
-      scale(this.scale*dirX, this.scale*dirY);
-      if (pInst._angleMode === pInst.RADIANS) {
+      // This line differs from p5.play upstream because GameLab users
+      // should be able to change the width and height of their animations.
+      scale(this.scale*dirX*this._horizontalStretch, this.scale*dirY*this._verticalStretch);
```

Comment:
Instead of adding the `_horizontalStretch` and `_verticalStretch` properties in p5.play.js, can we just track these in `GameLabSprite` (the only place we modify them)?  Then if we extract the transform methods here to a helper, and override the helper from GameLabSprite, we should be able to swap in our behavior change.

Sorry if you were already working on this, I know this might be a WIP PR.